### PR TITLE
fix(microsite): make the anchor link icon show up

### LINKED ISSUE #3707: microsite: make the anchor link icon show up
## Expected Behavior

When hovering over headings on the microsite (https://backstage.io), there should appear a little anchor link to the left. This is how it looks in GitHub when rendering the same page:

Screenshot 2020-12-14 at 13 19 52

## Current Behavior

Screenshot 2020-12-14 at 13 18 41

## Possible Solution

CSS Wizardry of level 3 or higher.

## Steps to Reproduce

1. Go to https://backstage.io/docs/overview/what-is-backstage
2. Hover over any heading, for example Benefits
3. Notice that there's no link icon on the left side, visually, but that it actually exists and is clickable etc (but black so it can't be seen on top of the background)

### PR DESCRIPTION
**Pull request by AdnCodez**
## Hey, I just made a Pull Request!

Closes #3707 

Added some CSS in main.css, to fix the color of the anchor on hover effect of each H2.
`h2:hover .hash-link {
  opacity: 1
}

.hash-link {
  fill: white;
}`

#### :heavy_check_mark: Checklist

scst1
scst2

### CODE REVIEW

**Comment by AdnCodez** [comment]
I hope this will be the last one, @freben

**Comment by OrkoHunter** [comment]
@AdnCodez

1. Please run `yarn prettier --write microsite/static/css/custom.css` to fix the failing build.
Then do, `git add -A` to stage the new file, and `git commit --amend` to update your previous commit, then do `git push -f origin master` to push your changes.

That will update this existing PR.

2. You don't have to do this now, but try to create a new branch when you are creating a new PR. So, don't use master next time for your next PR. :)

3. I updated your description to add `Closes #3707`. This tells GitHub to close that issue when this PR is merged.

**Comment by AdnCodez** [comment]
I have pretty much done that, and it didn't pass below
Screenshot from 2020-12-17 11-23-04
I will try what you mentioned thanks a lot

File: microsite/static/css/custom.css [hunk 1]
```diff
@@ -48,6 +48,14 @@ h6 {
   color: $textColor;
 }
 
+h2:hover .hash-link {
+  opacity: 1
```
Comment (freben) [inline microsite/static/css/custom.css:52]: yep there's a missing semicolon here

Comment (AdnCodez) [reply microsite/static/css/custom.css:52]: I don't know why that didn't get picked when I ran the command locally, my bad

**Review by freben — APPROVED** [review]
yepyep, now it passes! 🎉

**Pull request merged**